src/main/java/org/jenkinsci/plugins/ownership/security/folderspecific/FolderSpecificSecurity.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2013 Oleg Nenashev, Synopsys Inc.
```

Comment:
If you lend a significant part of the code, please use `2013-2018 Oleg Nenashev, Synopsys Inc, Dmitry Gruzd` or so